EduNLP/Formula/Formula.py

```diff
@@ -29,6 +29,10 @@ class Formula(object):
     <Formula: x>
     >>> f.elements
     [{'id': 0, 'type': 'mathord', 'text': 'x', 'role': None, 'var': 0}]
+
+    Returns
+    --------
+    the parsed abstracted syntax tree
```

Comment:
I think , for object definition, we don't  define the return in comments.
Besides, considering there are some attributes( like ast, elements, etc. ) in our object,maybe it is a good way to define the Attributes in comments, like this
https://github.com/RaRe-Technologies/gensim/blob/919b4154a5696a544b87f792992f25f5e4d59d3e/gensim/models/word2vec.py#L365